com.unity.multiplayer.mlapi/Runtime/Serialization/BitReader.cs

```diff
@@ -186,9 +184,10 @@ public object ReadObjectPacked(Type type)
             {
                 ulong networkId = ReadUInt64Packed();
 
-                if (networkingManager.SpawnManager.SpawnedObjects.ContainsKey(networkId))
+                var netManager = SpawnManager.SpawnedObjectsByNetworkingManager[networkId];
```

Comment:
After talking to @TwoTenPvP, I've come to see this is probably not correct code.  Instead, I intend to remove this table and instead pass the NM into ReadObjectPacked.